resources/leiningen/new/luminus/core/project.clj

```diff
@@ -14,9 +14,6 @@
   :main ^:skip-aot <<project-ns>>.core
 
   :plugins [<% if plugins %><<plugins>><% endif %>] <% if cucumber-feature-paths %>
-  ;; Exclude :no-ns-form-found linter to avoid warnings on step definitions.
-  ;; This can be done per step file once https://github.com/jonase/eastwood/issues/165 is done.
```

Comment:
Said issue is fixed with https://github.com/jonase/eastwood/tree/v1.2.4#ignored-faults

IDK if by default one would get any issue at all. If that's the case you would want to template `:eastwood {:ignored-faults ,,,` accordingly